closed/src/java.base/share/native/libjncrypto/NativeCrypto.c

```diff
@@ -917,6 +935,11 @@ JNIEXPORT jlong JNICALL Java_jdk_crypto_jniprovider_NativeCrypto_DigestCreateCon
     }
 
     return (jlong)(intptr_t)context;
+
+    releaseContexts:
+        printErrors();
+        Java_jdk_crypto_jniprovider_NativeCrypto_DigestDestroyContext(env, thisObj, (jlong)(intptr_t)context);
+        return -1;
```

Comment:
Please fix the indentation here (the label starts at the beginning of the line, other lines should be indented normally.

Reply:
Indentation was fixed.